site/global.d.ts

```diff
@@ -0,0 +1,5 @@
+declare global {
```

Comment:
This looks like overhead. Consider to just add a type in the component module, like
```ts
type Content = {
  title: string
  description: string
  icon: string
}
...
content.reduce<[Array<Content>, Array<Content>]>
```